src/SiteConfig/GrabySiteConfigBuilder.php

```diff
@@ -125,6 +126,20 @@ protected function processExtraFields($extraFieldsStrings)
         return $extraFields;
     }
 
+    /**
+     * Processes http_header(*) config, prepending HTTP_ string to the header's name.
+     * See : https://github.com/symfony/browser-kit/blob/5.4/AbstractBrowser.php#L349.
+     */
+    protected function processHeaders(array $configHeaders): array
```

Comment:
with fresh new eyes I think this should not be done here, so that the SiteConfig kept original header names.
should be a private function LoginFormAuthenticator, what do you think?

Reply:
You're right, it's a bit weird to have it here.